lib/hcl/version.rb

```diff
@@ -1,3 +1,3 @@
 module HCl
-  VERSION = '0.4.15'
```

Comment:
I'd prefer the version bump to be in a separate commit, could you remove this change from the patch?